qchem/pennylane_qchem/qchem/__init__.py

```diff
@@ -18,14 +18,14 @@
 
 __all__ = [
     "read_structure",
-    "meanfield_data",
+    "meanfield",
     "active_space",
-    "decompose_hamiltonian",
+    "decompose_molecular_hamiltonian",
     "_qubit_operator_to_terms",

```

Comment:
I tried to understand these"undefined variable name.." messages from CodeFactor but could not interpreted which is the problem. Any ideas here?